Makefile

```
@@ -137,3 +148,26 @@ test-local-runtime:
 	source pulp-runtime/configs/pulp.sh; \
 	cd tests && ../pulp-runtime/scripts/bwruntests.py --proc-verbose -v --report-junit -t 600 --yaml -o simplified-runtime.xml runtime-tests.yaml
 
+
+# Bender integration
+
+VLOG_ARGS += -suppress 2583 -suppress 13314
+BENDER_BUILD_DIR = sim
+
+.PHONY: bender-script bender-build bender-rm
+bender-script: 
+	echo 'set ROOT [file normalize [file dirname [info script]]/..]' > $(BENDER_BUILD_DIR)/compile.tcl
+	./bender script vsim \
+		--vlog-arg="$(VLOG_ARGS)" --vcom-arg="" \
+		-t rtl -t test \
+		| grep -v "set ROOT" >> $(BENDER_BUILD_DIR)/compile.tcl
+
+bender: 
+ifeq (,$(wildcard ./bender))
+	curl --proto '=https' --tlsv1.2 -sSf https://iis-people.ee.ethz.ch/~andkurt/bender-init \
```

```suggestion
	curl --proto '=https' --tlsv1.2 -sSf https://fabianschuiki.github.io/bender/init \
```
The script is now officially and publicly available under this URL.